engine/src/main/java/org/terasology/persistence/serializers/PrefabSerializer.java

```diff
@@ -167,7 +156,7 @@ private void applyComponentChanges(Module context, EntityData.Prefab prefabData,
                     }
                 }
             } else if (componentData.hasType()) {
-                logger.error("Prefab contains unknown component '{}'", componentData.getType());
+                logger.warn("Prefab '{}' contains unknown component '{}'", prefabData.getName(), componentData.getType());
```

Comment:
question: I see you changed this from `error` to `warn`. Is it not an error to reference unknown components? What would be an example of this? (I see a lot of this sort of thing in the logs, but it doesn't seem to come up in #stabilization.)

Reply:
I believe it should be a warning, as we mention it but continue with normal execution.
It can be components that no longer exist or components that are not available in the selected set of modules. It could have the effect that some system logic is not executed because event listeners expect a component to be present, but that would be desired behaviour of our modular ECS architecture I think.

Reply:
This sounds plausible. I don't know much about how optional dependencies and interactions work in this architecture.

I worry that continuing with normal execution teaches people to ignore warnings and makes it harder to find what's relevant when hunting down problems.

but whether to continue execution is well outside the scope of this PR, and I guess I can agree that if we're willing to continue execution it is more like a "warning" than an "error."

Reply:
I think continuing execution is "usually" what we want to do - we should not hard-crash the game (or drop back to menu) too easily imho. 

I think the current setup does not take into account that some component may come from an optional dependency, and thus does not need to be there in all cases.. It would be nice to error out on expected components being unavailable, but being silent about cases where this is not a big deal... way beyond the scope of this, as said. 

I'm wondering whether there's something about that in here: https://github.com/MovingBlocks/Terasology/pull/3236